packages/deprecated/markdown/package.js

```diff
@@ -11,7 +11,7 @@ Package.onUse(function (api) {
   api.versionsFrom('3.0-alpha.6');
   api.use('ecmascript@1.0.0-alpha300.6');
   api.use("templating@2.0.0-alpha300.6", "client", {weak: true});
-  api.mainModule('template-integration.js', 'client', { lazy: true });
```

Comment:
It would be nice if we could leave this module as lazy since it allows the package to be dynamically imported. Do you have any ideas on why it didn't work? Do you know if the tests passed with Meteor 2.x?

Reply:
No ideia, maybe related to bundler? Could you try to check this @zodern ?

Reply:
I'll look into it. There were a few changes that could affect this.

Reply:
@zodern in version 2.x works properly. 

I was trying to understand what's happening and it looks like what's being generated by the linker for the `markdown` package is:

```
Package["core-runtime"].queue("markdown"), [..deps], function () {
   return () {}
}
```

That's as far as I got.

Reply:
Maybe we could merge this and open an issue related to meteor 3 to investigate this further

Reply:
I would be fine with that.

Reply:
@zodern could you approve the PR then?